charts/modules/apps/crossplane/values.yaml

```diff
@@ -430,12 +430,12 @@ crossplane-aws-iam:
   RolePolicyAttachment:
     items:
       provider-aws-adminaccess:
```

Comment:
This is where the comment for review late could be added.

Reply:
Done